client/src/pages/Register/PaymentStep.tsx

```diff
@@ -98,68 +158,89 @@ function PaymentStep(props: RegisterStepProps) {
   const submitPayByCheck = () => {
     setLoading(true);
 
-    props.submitPayment({
-      paymentType: 'Check',
-      paymentData,
-    });
+    processResult('Check');
   };
 
+  const onDepositChange = ({ formData }: { formData: { deposit: string  } }) => {
+    try {
+      setDepositChoice(formData);
+      const values = JSON.parse(formData.deposit);
+
+      setTotal(jsonLogic.apply(values.logic, totals));
+
+    } catch (e) {
+      console.error('deposit logic error', e);
+    }
+
+  }
+
+  const payPalOptions: (PayPalScriptOptions | undefined) = config.payPalOptions;
+
+  if (process.env.NODE_ENV === 'development' && !!payPalOptions && !payPalOptions['client-id']) {
+    payPalOptions['client-id'] = 'sb';
+  }
+
   return (
-    <div className="payment-form">
-      <h1>Choose your payment option</h1>
-      <p>
-        If you need to go back and change a value, refresh this page and it
-        will take you back to the registration form with your values filled
-        in
-      </p>
```

Comment:
The back button now works as one would expect.  TODO: decide that when you hit the FinalStep, perhaps we should make it so one can't go back?